ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -643,6 +643,17 @@ impl IntegerType {
         self.num_bits
     }
 
+    /// Return the number of bytes required to store this integer type.
```

Comment:
> It's the size assuming the integer is aligned

I don't think we can conflate the two concepts like this, because the point of alignment is that it's an "at most" rather than an "at least" thing. In other words, if something has 8 byte alignment and is 8 bytes long, but you want to store it at an address that is 8 byte aligned, you only need 8 bytes to store it, not 16.

Reply:
I'm now confused. You can't specify the alignment of a type in this way, because it depends where you store it.

Reply:
We just need to know how big the thing is so we can use an appropriately-sized loads and stores. Alignment isn't relevant for this part of the code.

Reply:
Try https://github.com/ykjit/yk/pull/991/commits/2253bd2ee6d5e1be539d824cf841173a9179ae83

Reply:
I dunno if I have a broken view of things, but padding for alignment doesn't really contribute to the size of type. In my view padding comes into play when you decide where to store an instance of a type, so it doesn't make much sense to talk about the alignment of a *type*.

Reply:
I'm an idiot and misread your comment! Sorry, "size assuming it's aligned" is fine.

Reply:
It's OK as-is.